restaurants/models.py

```diff
@@ -0,0 +1,263 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+# Create your models here.
+
+
+class BaseModel(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    deleted_at = models.DateTimeField(auto_now_add=True)
+    created_by = models.ForeignKey(
+        "CustomUser", on_delete=models.DO_NOTHING, null=True, blank=True
+    )
+    updated_by = models.ForeignKey(
+        "CustomUser", on_delete=models.DO_NOTHING, null=True, blank=True
+    )
+    deleted_by = models.ForeignKey(
+        "CustomUser", on_delete=models.DO_NOTHING, null=True, blank=True
+    )
+    is_acitve = models.BooleanField(default=False)
+
+    class Meta:
+        abstract = True
+
+
+class CustomUser(BaseModel, AbstractUser):
+    """
+    This model represents Users of app.
+
+    Attributes:
+
+    first_name(CharField): The first name of user
+    last_name(charField): Last name of user
+    email(EmailField): Email of user
+    contact_number(IntegerField): Mobile number of user
+    type(Enum): User type like Customer, restaurant owner ,admin or rider(driver)
+
+    """
+
+    TYPE = (
+        ("MANAGER", "MANAGER"),
+        ("CUSTOMER", "CUSTOMER"),
+        ("ADMIN", "ADMIN"),
+        ("OWNER", "OWNER"),
+        ("RIDER", "RIDER"),
+    )
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+    email = models.EmailField()
+    contact_number = models.IntegerField()
+    type = models.CharField(max_length=255, choices=TYPE, default="CUSTOMER")
+
+    def __str__(self):
+        return self.first_name + " " + self.last_name
+
+    class Meta:
+        db_table = "user"
+
+
+class Location(BaseModel):
+    """
+    This model represents location of the restaurant
+
+    Attributes:
+
+    address(CharField): Proper/exact address like flat number,society name,floor , Office number etc.
+    latitude(FloatField): Used to fetch location
+    longitude(FloatField): Used to fetch location
+
+    """
+
+    address = models.CharField(max_length=255)
+    latitude = models.FloatField()
+    longitude = models.FloatField()
+
+    class Meta:
+        db_table = "restaurant_location"
+
+
+class Image(BaseModel):
+    """
+    This model represents Images for menu or bank images of restaurant
+
+    Attributes:
+
+    image(FileField): Upload multiple images for the menu or bank account images
+    """
+
+    image = models.FileField(upload_to="Documents/")
+
+    class Meta:
+        db_table = "restaurants_menuimage"
```

Comment:
This is incorrect. I can see you are using this model for bank images as well, but here it says `restaurant_menuimage` which is confusing for the database administrator. Please talk to Ravindra bhai, I already explained him how we should split the different doc types.